module4/projects/solo_stretch_tech/index.md

```diff
@@ -3,14 +3,137 @@ title: Solo Stretch Tech
+- BE Planning Docs: We also expect to see a strong schema design before you start coding. Use any tool you'd like for this. We want to see what data you plan to manage, how it will be organized, and what the relationships between the data will be. 
+- Stretch Tech Learning Resources: Please find at least 2 resources per stretch tech (4 total) that you will use for learning. This could be the official docs, a tutorial, a YouTube series, etc. 
+</section>
+
+<section class="dropdown">
+### Day 7
+
+- Repo(s): Please create your repo(s) and link them in the google sheet. You don't need to have any code in there yet.
+- Project Board: You will link your project board, which should include:
+  - At least 4 columns, i.e. `To Do`, `In Progress`, `In Review`, `Merged/Done`
+  - At least 6 tickets, with each ticket including a user story
+  - All tickets should be converted to **[issues](https://docs.github.com/en/issues)** in your repo
+      - Issues are sometimes disabled. To enable them go to Settings in the upper right corner, then scroll down to the Features section. Check the Issues box to enable them.
+  - (optional) The use of **[labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels)** could be very helpful in organizing your Project Board
+    - To view labels on the project board, click the down carrot on the View tab (top left of page) > Fields > ✔️ Labels.
+</section>
+
+<section class="dropdown">
+### Day 9
+
+- First PR: We'd love to (finally) see some code! Choose any ticket from your project board and get it done! Push up a PR and drop the link in the google sheet. This can be BE or FE code. Your PR should [close the associated ticket by referencing the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue). You can merge this PR anytime, we will not be doing a code review.
+</section>
+
+
+<section class="dropdown">
+### Day 10
+
+- Presentation Slides: Link your presentation slides here. More details in the "Presentation Details" section.
+</section>
+
+## Presentation Details
+
+### Purpose
+Your presentation will highlight your progress on theis project and your plans for your job search moving forward. It’s an opportunity to reflect on your experiences and demonstrate how you’ve grown and learned throughout the process.
+
+### Structure
+- Student-directed presentation (~10-15 minutes)
+  - Develop an organized slide deck or Google document to present your progress and plan moving forward.
+  - Include visual aids, such as charts or graphs, to enhance your presentation.
+- Instructor feedback (~5 minutes)
+
+### Potential Outcomes
```

Comment:
Do we need a rubric? Or is this sufficient?

Reply:
I'm leaning yes. It's listed as a graduation requirement that a student passes the stretch tech project, so it should probably be pretty clear what "passing" looks like. Right now this reads as being focused only on the quality of the presentation.

Reply:
Rubric category ideas:
* Presentation
* Work Ethic (meets due dates)
* Preparation (completes deliverables thoroughly/as we intended)